### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by anniviz**
Acceptance criteria:
- The user can switch days in a calendar
- Plans in the past can be edited
- Selected date is shown in header

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,40 +1,43 @@
...
+  const today = new Date()
+  const [selectedDay, setSelectedDay] = useState(today)
+  const selectedDayString = `${selectedDay.getFullYear()}-${selectedDay.getMonth()}-${selectedDay.getDate()}`
 
-  const [medicationToEdit, setMedicationToEdit] = useState({})
+  const {
+    activeMedications,
+    medicationToEditId,
+    setMedicationToEditId,
+    handleSubmit,
```
Comment (Morten-Moeller) [inline src/App.js:16]: Consider renaming this. I cant assume what it does.

File: src/components/Header.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 
-export default function Header() {
-  const today = new Date()
+Header.propTypes = {
+  selectedDay: PropTypes.instanceOf(Date),
+}
 
-  return <HeaderStyled>{formatDate(today)}</HeaderStyled>
+export default function Header({ selectedDay }) {
+  return <HeaderStyled>{formatDate(selectedDay)}</HeaderStyled>
```
Comment (Morten-Moeller) [inline src/components/Header.js:9]: Consider exporting an styled.component and hand the already formatted date as a child.

File: src/pages/MedicationPage.js [hunk 3]
```diff
@@ -69,31 +90,27 @@ export default function MedicationPage({
...
-  }
-
   function handleEditClick(id) {
-    const index = medications.findIndex(medication => medication.id === id)
-    setMedicationToEdit(medications[index])
+    setMedicationToEditId(id)
     setActivePage('form')
   }
 }
 const Grid = styled.div`
```
Comment (Morten-Moeller) [inline src/pages/MedicationPage.js:108]: Please have a look if not a section is the right element.

**Review by Morten-Moeller — APPROVED** [review]
Very nice code, and good improvement using custom hooks.
Keep on Rolling!

File: src/pages/MedicationPage.js (hunk 3 above)
Comment (anniviz) [reply src/pages/MedicationPage.js:108]: the grid contains the whole page, header included, so basically the whole body. in my understanding a div is the right element here.

**Review by LisanneArdern — COMMENTED** [review]
Looks good to me! Even though it does seem like a complicated way for such a simple thing the code is still easy to read :D

**Pull request merged**